postgis/templates/deployment.yaml

```diff
@@ -34,7 +34,9 @@ spec:
             - name: POSTGRES_USER
               value: {{ .Values.postgres_user | quote }}
             - name: POSTGRES_PASSWORD
-              value: {{ .Values.postgres_password | quote }}
+              value: {{ .Values.postgres_password }}
```

Comment:
Is there a reason your moved the `quote` from postgres_password? I put it there so that we can have passwords with special characters.